js/components/controlBar.js

```diff
@@ -106,7 +107,7 @@ var ControlBar = React.createClass({
     evt.preventDefault();
     if (this.props.controller) {
       this.props.controller.toggleFullscreen();
-      if (this.vr && this.isMobile && this.props.controller.isVrStereo) {
+      if (this.vr && this.isPhone && this.props.controller.isVrStereo) {
```

Comment:
We already have a `Utils.isIPhone()` helper function. Could we use that here instead?

Reply:
@jpfranco no, I told about any phones (not only iPhones.)

Reply:
Oh ok, sorry. I read that wrong.